schema_utils.py

```diff
@@ -229,47 +261,113 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: list(str). An object.
```

Comment:
list(*) -- you're checking the length, the contents don't matter.

Reply:
Done